astropy/io/fits/diff.py

```diff
@@ -249,10 +250,23 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
             are kept in memory or output.  If a negative value is given, then
             numdiffs is treated as unlimited (default: 10).
 
-        tolerance : float, optional
+        rtol : float, optional
             The relative difference to allow when comparing two float values
             either in header values, image arrays, or table columns
-            (default: 0.0).
+            (default: 0.0). Values which satisfy the expression
+
+                abs(x1 - x2) > atol + abs(x1)*rtol
+
+            are considered to be different.
+
+        atol : float, optional
+            The absolute difference to allow when comparing two float values
```

Comment:
This parameter than also needs a `.. versionadded:: 1.3` (assuming this goes into 1.3).